src/braze/tags/connectedContent.ts

```diff
@@ -129,7 +129,7 @@ export default <ITagImplOptions>{
       }
       ctx.environments[this.options.save || 'connected'] = jsonRes
     } catch (error) {
-      return res.body
+      console.error(`Could not parse response body: "${res.body}"`)
```

Comment:
This is for handling an API that returns plain text instead of JSON response. What you are trying to fix is an API that returns 500 error. I think we are mixing a few scenarios here:

1. API returns 200 plain text
2. API returns 200 JSON
3. API returns 500 plain text
4. API returns 500 JSON

Since I no longer have a braze account, could you help to double check how Braze behaves under each of the above scenarios?

I feel we should be checking status_code before trying to parse the JSON body here.

Reply:
Gotcha. That makes sense. I'll investigate and push some updates.

Reply:
Findings:

`text/plain`
200: No save, Yes print
500: No save, No print

`application/json`
200: Yes save, No print
500: No save, No print

Reply:
Another finding is that any parsable JSON will function now how JSON Objects used to function. You'll see that update reflected in my changes, as well as in the unit tests.